### PR TITLE
Polish the workaround with the fix in salt module.

### PR DESCRIPTION
**Pull request by nick-wang**
The corresponding [fix](https://github.com/SUSE/salt-shaptools/commit/dc0b1f5afebfe9a14892a64b5f1be4a675d8fa53) released in salt-shaptools-0.2.8

### CODE REVIEW

**Comment by nick-wang** [comment]
With the fix in salt-shaptools, the hard code of 60 secs for waiting I/O complete is removed. 

However, the packet send from sync source to sync target is not included, but 5 secs is long enough for an acknowledge.

File: drbd-formula.spec [hunk 1]
```diff
@@ -19,14 +19,14 @@
...
-Version:        0.3.9
+Version:        0.3.10
 Release:        0
 Summary:        DRBD deployment salt formula
 License:        Apache-2.0
 URL:            https://github.com/SUSE/%{name}
 Source0:        %{name}-%{version}.tar.gz
 Requires:       drbd-utils
-Requires:       salt-shaptools
+Requires:       salt-shaptools >= 0.2.8
```
Comment (arbulu89) [inline drbd-formula.spec]: Just maybe you need to change this dependency version if you finally upgrade `salt-shaptools` in the last PR

**Review by arbulu89 — APPROVED** [review]
Thanks @nick-wang!
Anyway, I wouldn't be worried of having bigger sleeps if this provides stability, we are not really looking for optimization (the faster the better, that' true hehe).

**Comment by nick-wang** [comment]
> Thanks @nick-wang!
> Anyway, I wouldn't be worried of having bigger sleeps if this provides stability, we are not really looking for optimization (the faster the better, that' true hehe).

I see~ Actually, with the fix in `salt-shaptools` doesn't need to plus a extra seconds, since normally the packet will be delivered within second. I just add 5 more in case anything unexpected, also avoid the unlucky 13.

**Pull request merged**